.github/workflows/build.yml

```diff
@@ -27,9 +27,9 @@ on:
 
 env:
   # Use SHA instead of the branch for caching purposes
-  MONO_TAG: mono-6.8.0.123
+  MONO_TAG: mono-6.12.0.102
   PYTHON_VERSION: 3.8
-  EMSDK_VERSION: 1.38.47-upstream
+  EMSDK_VERSION: 1.39.9
```

Comment:
The version that matches `mono-6.12.0.102` is Emscripten 1.39.7, but I included a patch which adds support for 1.39.9 as 1.39.7 is broken (doesn't generate `.wasm` file for the Mono build).